integrationtests/python_src/px4_it/mavros/mavros_offboard_yawrate_test.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python2
```

Comment:
Does it really need to be Python 2 still?

Reply:
I can check with python 3, it should work fine as long as melodic does work fine with it. Will let you know.

Reply:
@julianoes I tried chaning to python 3 but was immediately put off by 
```
Traceback (most recent call last):
  File "/home/roque/Git/PX4/Firmware/integrationtests/python_src/px4_it/mavros/mavros_offboard_yawrate_test.py", line 44, in <module>
    import rospy
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/__init__.py", line 49, in <module>
    from .client import spin, myargv, init_node, \
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/client.py", line 52, in <module>
    import roslib
  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslib/__init__.py", line 50, in <module>
    from roslib.launcher import load_manifest  # noqa: F401
  File "/opt/ros/melodic/lib/python2.7/dist-packages/roslib/launcher.py", line 42, in <module>
    import rospkg
ModuleNotFoundError: No module named 'rospkg'
```
I think Melodic uses python2 by default, so maybe we keep at 2 and move to 3 when we move all the compatibility things to the next ROS LTS (which should be Python 3 compatible), otherwise we would have to build ROS with forced python 3, ie, not the default one.

Reply:
Ok, right.